### ISSUE TITLE
force_recompile is ignored

### ISSUE THREAD

**Issue by TheGeorge** [tsloughter/rebar3_erlydtl_plugin#15]
Since needs_compile() is used exclusively to check if a file has changed the force_recompile option in compiler_options is ignored.

**Comment by tsloughter** [comment]
Ah, yes, thanks, that needs to be changed.

**Comment by TheGeorge** [comment]
I don't think that you need that check there in the first place. 

erlydtl itself only recompiles the templates if they have changed if the force_recompile option is not set.

**Comment by tsloughter** [comment]
Good point. I'll try to remember to take that out. I need to figure out why erlydtl 0.12.0 hex package doesn't work properly so I can fix up the plugin and publish, will include this change as well when I do.
